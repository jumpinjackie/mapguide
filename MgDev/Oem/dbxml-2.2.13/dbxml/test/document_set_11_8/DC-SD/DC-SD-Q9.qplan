<XQuery>
  <Navigation>
    <QueryPlanFunction result="collection" container="11.8.2.dbxml">
      <OQPlan>V(node-attribute-equality-string,@id,=,'I5')</OQPlan>
    </QueryPlanFunction>
    <Step axis="child" name="catalog" nodeType="element"/>
    <Step axis="child" name="item" nodeType="element"/>
    <DbXmlFilter>
      <Navigation>
        <Step axis="attribute" name="id" nodeType="attribute"/>
        <DbXmlCompare name="equal">
          <Sequence>
            <AnyAtomicTypeConstructor value="I5" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
          </Sequence>
        </DbXmlCompare>
      </Navigation>
    </DbXmlFilter>
    <Step axis="descendant" name="ISBN" nodeType="element"/>
    <Step axis="child">
      <ItemType testType="text"/>
    </Step>
  </Navigation>
</XQuery>
