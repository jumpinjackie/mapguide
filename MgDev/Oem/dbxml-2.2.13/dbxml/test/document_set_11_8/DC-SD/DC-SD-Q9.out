5993120826517